content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -46,6 +46,14 @@ localization:
 
 .. _india/e-invoicing:
 
+Indian Configuration
+====================
+
+In :menuselection:`Settings --> Users & Companies --> Companies`, add your PAN and GST number.
+
+.. image:: india/company-form.png
+   :alt: Company form
```

Comment:
```suggestion
```

Reply:
We try to limit the number of screenshots and only include them when they're really necessary, so I suggest removing this one (don't forget to delete the actual file as well).